lib/ts/recipe/session/sessionRecipe.ts

```diff
@@ -183,7 +178,7 @@ export default class SessionRecipe extends RecipeModule {
                 refreshTokenValidity: response.refreshTokenValidity,
             };
             if (this.checkIfInServerlessEnv()) {
-                await storeHandshakeInfoInFile(this.handshakeInfo);
+                storeIntoTempFile(HANDSHAKE_INFO_FILE_PATH, this.handshakeInfo);
```

Comment:
Arent you supposed to to do JSON.stringify here?

Reply:
No, done by the function storing in the file.